cli/src/cluster_query.rs

```diff
@@ -84,6 +84,13 @@ impl ClusterQuerySubCommands for App<'_, '_> {
                         .takes_value(true)
                         .help("Stop after submitting count transactions"),
                 )
+                .arg(
+                    Arg::with_name("lamports")
+                        .long("lamports")
```

Comment:
In courtesy of the `ping`, I wondered what corresponding short options is, but couldn't determine. So, there is no `short` by intention. Maybe `ping`'s `-p pattern` or `-s packetsize` will be the closest counterpart?

Reply:
No short option here is fine with me.

Reply:
I see! Me too.